script/start

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+# Run single command from Procfile
+#
+# This script is a basic replacement for foreman when running on Docker. When
+# starting the docker instance, specify as command `script/start <type>`.
+# `type` is looked up in `Procfile` and the command is started. This allows e.g.
+# docker-compose to run multiple process/dyno types from a single image, without
+# needing to know the exact command (which may change over time).
+#
+if [ ! "$1" ]; then
+  echo "Usage: $0 <process type>" 1>&2
+  exit 1
+fi
+
+PROCFILE=`dirname "$0"`/../Procfile
```

Comment:
Could you put the start script into the same folder as the Procfile? I do not like this kind of relative file paths and I also do not see much need for the `dirname`, which just makes the code more difficult to read.

Reply:
This is ugly, I agree. It should be just `Procfile` (or overridable with a `-f` option like [foreman](https://ddollar.github.io/foreman/)). It is assumed anyway that the script is run from the app's root, so it can still bein script/. Or do you still prefer it in the root?

Reply:
i'd put both files into root, but it's not a real issue for me